tests/beaker/lib/lvm_helper.rb

```diff
@@ -0,0 +1,91 @@
+# Verify if a physical volume, volume group, logical volume, or filesystem resource type is created
+#
+# ==== Attributes
+#
+# * +resource_type+ - resorce type, i.e 'physical_volume', 'volume_group', 'logical_volume', or 'filesystem'
+# * +resource_name+ - The name of resource type, i.e '/dev/sdb' for physical volume, vg_1234 for volume group
+#
+# ==== Returns
+#
+# +nil+
+#
+# ==== Raises
+# assert_match failure message
+# ==== Examples
+#
+# verify_if_created?(agent, 'physical_volume', /dev/sdb')
+def verify_if_created?(agent, resource_type, resource_name, vg=nil, properties=nil)
```

Comment:
The method signature is confusing to use. Consider instead using the "opts={}" hash convention for Ruby.